Core/src/Material/Interactions.cpp

```diff
@@ -75,95 +102,107 @@ inline float logDeriveWMax(float mass, float qOverP,
 /// where (Z/A)*rho is the electron density in the material and x is the
 /// traversed length (thickness) of the material.
 inline float computeEpsilon(float molarElectronDensity, float thickness,
-                            const Acts::RelativisticQuantities& rq) {
+                            const RelativisticQuantities& rq) {
   return 0.5f * K * molarElectronDensity * thickness * rq.q2OverBeta2;
 }
 
 /// Compute epsilon logarithmic derivative w/ respect to q/p.
-inline float logDeriveEpsilon(float qOverP,
-                              const Acts::RelativisticQuantities& rq) {
+inline float logDeriveEpsilon(float qOverP, const RelativisticQuantities& rq) {
   // only need to compute d(q²/beta²)/(q²/beta²); everything else cancels.
   return 2 / (qOverP * rq.gamma * rq.gamma);
 }
 
 /// Compute the density correction factor delta/2.
 inline float computeDeltaHalf(float meanExitationPotential,
                               float molarElectronDensity,
-                              const Acts::RelativisticQuantities& rq) {
+                              const RelativisticQuantities& rq) {
   /// Uses RPP2018 eq. 33.6 which is only valid for high energies.
   // only relevant for very high ernergies; use arbitrary cutoff
   if (rq.betaGamma < 10.0f) {
     return 0.0f;
   }
   // pre-factor according to RPP2019 table 33.1
-  const auto plasmaEnergy =
+  const float plasmaEnergy =
       PlasmaEnergyScale * std::sqrt(1000.f * molarElectronDensity);
   return std::log(rq.betaGamma) +
          std::log(plasmaEnergy / meanExitationPotential) - 0.5f;
 }
 
 /// Compute derivative w/ respect to q/p for the density correction.
-inline float deriveDeltaHalf(float qOverP,
-                             const Acts::RelativisticQuantities& rq) {
+inline float deriveDeltaHalf(float qOverP, const RelativisticQuantities& rq) {
   // original equation is of the form
   //     log(beta*gamma) + log(eplasma/I) - 1/2
   // which the resulting derivative as
   //     d(beta*gamma)/(beta*gamma)
   return (rq.betaGamma < 10.0f) ? 0.0f : (-1.0f / qOverP);
 }
 
-}  // namespace
+inline float _computeEnergyLossLandauFwhm(const Acts::MaterialSlab& slab,
```

Comment:
Consider using a `detail` namespace (even if it's just in the `.cpp`) instead of prepending the `_` to disambiguate the function names.